gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/workunit/packer/KafkaTopicGroupingWorkUnitPacker.java

```diff
@@ -338,7 +338,10 @@ private Double getMinWorkUnitSize(WorkUnit workUnit) {
    */
   @VisibleForTesting
   static double getContainerCapacityForTopic(List<Double> capacities, ContainerCapacityComputationStrategy strategy) {
-    Preconditions.checkArgument(capacities.size() > 0, "Capacities size must be > 0");
+    //No previous stats for a topic? Return default.
```

Comment:
Just curious how did we deal with the new topics previous? Asking because capacities for new topics should always be null?

Reply:
This was only tested with topics which already had some stats. This config has not been enabled yet in a large scale production environment at LI yet. On a recent test, this issue was uncovered.